src/Fixer/Strict/DeclareStrictTypesFixer.php

```diff
@@ -53,7 +53,7 @@ public function getPriority(): int
 
     public function isCandidate(Tokens $tokens): bool
     {
-        return isset($tokens[0]) && $tokens[0]->isGivenKind(T_OPEN_TAG);
+        return null !== $this->getOpenTagIndex($tokens);
```

Comment:
would it work if we use `Tokens::isMonolithicPhp()` here instead?

Reply:
It can be used, done.